front/src/components/member/MemberSidebar.vue

```diff
@@ -83,6 +84,18 @@ export default {
     isGuest() {
       return this.$store.getters["member/isGuest"];
     },
+    shortNickname() {
+      return Array.from(this.member.nickname)
+        .slice(0, NICKNAME_LENGTH)
+        .join("")
+        .trim();
+    },
+    nicknameTail() {
+      if (Array.from(this.member.nickname).length > NICKNAME_LENGTH) {
+        return "...";
+      }
+      return "";
```

Comment:
```suggestion
      return this.member.nickname.length > NICKNAME_LENGTH ? "..." : "";
```

특수문자, 이모티콘을 제외했으니 Array로 변환하는 과정이 필요하지 않을 것 같아요.😄